tests/components/generic/conftest.py

```diff
@@ -89,8 +93,8 @@ def mock_create_stream(hass: HomeAssistant) -> _patch[MagicMock]:
     )
 
 
-@pytest.fixture
-async def user_flow(hass: HomeAssistant) -> ConfigFlowResult:
+@pytest.fixture(name="user_flow")
+async def user_flow_fixture(hass: HomeAssistant) -> ConfigFlowResult:
```

Comment:
This doesn't change a whole lot tbf

Reply:
Yeah, it was just to match the naming of the other one. But not necessary.

Reply:
My impression is the one thing this style helps with is making it harder to accidentally call the function `user_flow` from another function, e.g. when you forget to add as an input fixture to the other function. There may be linters that catch this though from what I remember, but I have found myself missing this up a couple times.

Reply:
Yes, sometimes I've accidentally done that. Naming it 'xxx_fixture' makes it a bit more obvious I find.